frontend/app/dtos/parse-address-request-dto.server.ts

```diff
@@ -0,0 +1,33 @@
+export interface ParseWSAddressRequestDTOProps {
+  addressLine: string;
+  city: string;
+  province: string;
+  postalCode: string;
+  country: string;
+  language: string;
+}
```

Comment:
While I'm not completely against having a separate DTO folder in the project, it doesn't really feel right to me.

Types that are returned by the wsaddress service should be defined in the wsaddress service. This feels move js-conventional to me.

Reply:
Moved DTO to wsaddress-service